talkideas.md

```diff
@@ -8,3 +8,39 @@ In-depth debugger talk
+
+We have a responsibility for developing the technical skills and experience of developers, for building a better profession and for creating a healthy environment to work in. How can we shape careers that challenge and value developers, and how do you identify and respond when things go wrong?
+
+This talk will use my insights from building career paths from scratch at emerging companies, and examine the successes and failures that are common across the industry.
+
+
+Modular CSS with WebPack
+========================
+Composition has long been a goal of CSS design. Creating a rich library of styles that can be reused and combined saves time in the long run, but often lead to unmaintainable or verbose code.
+
+WebPack is commonly used for bundling JS modules, but its flexibility means it can also be used to create modular stylesheets: local styles that can be imported and exported just like scripts.
+
+This talk introduces WebPack and will demonstrate how to write, bundle and use CSS modules. We'll also examine the benefits of using WebPack, and why it can help in ways other tools have failed in the past.
+
+
+Dispelling Devops with Ansible
+==============================
+Do you fear that owning your own web server is reserved for the elite and all-powerful society of sysadmins? Then fear not, the fact is that it's never been easier.
+
+Ansible is a powerful tool that lets you control servers by configuration instead of command. It's designed for ease-of-use, and this talk will show how installing software, automating tasks and setting up access control is managed using its simple YAML interface. We'll use these lessons to set up a basic website on a fresh server, and explain some of the fundamentals of hosting a site along the way.
+
+- This talk is aimed at hosting beginners.
+- The talk will features some command-line usage, but user-friendly tools will be the major theme.
+- I want to demonstrate Ansible alongside other tools designed for ease-of-use, such as Digital Ocean and the Caddy webserver.
+
+
+Sustainable Open Source
+=======================
+Two years ago, I became the maintainer of a successful open source project. Its success was a surprise to me more than anyone, and I've faced burnout several times in the course of its development. It has, however, been an invaluable glimpse into the best and worst parts of open source.
+
+This story is important now more than ever, because there are thousands of developers writing software that we are coming to rely on in our work. What are the forces acting on us that cause stress and burnout? How can we look after those in our community donating their time and effort to projects? Is it time to re-examine the relationship we have with free software?
+
```

Comment:
I like this one too. However I feel that there could be more information as to what the talk will contain. Lots of unanswered questions, which if I saw it already in a schedule, I might go along to out of curiousity. But I might not vote for it, as it sounds a bit hand wave-y and I'm not sure what the talk would consist of.

With Remy's talk on the Toxic Side of Free, you knew what the project was, and you had a couple of examples, even from the very short brief from AsyncJS

```
Remy Sharp adorns Async for the first time evar to talk about the dark side of running JS Bin as an open for all playground.

Tales of spam, abuse, police, terrorists and VAT.
```